src/org/usfirst/frc/team2706/robot/controls/StickRumble.java

```diff
@@ -0,0 +1,87 @@
+package org.usfirst.frc.team2706.robot.controls;
+
+import edu.wpi.first.wpilibj.GenericHID.RumbleType;
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.command.Command;
+
+import org.usfirst.frc.team2706.robot.Robot;
+
+public class StickRumble extends Command{
+    
+    protected Joystick joystick;
+    protected Joystick operatorJoy;
```

Comment:
Our code format usually puts spaces between each of the initialization variables